### DISCUSSION TITLE
iiAB short intro YouTube video

### DISCUSSION THREAD

**Discussion by crconsultants** [iiab/iiab#3001] [Show and tell]
We created this YouTube with a short first impression introduction to the Medical emergency use of IIAB after my successful installation of a test site on a laptop using Ubuntu.  There didn't seem to be much out there like this.
Perhaps you want to add it to the YouTube video's highlighted?

https://youtu.be/-FAw27pOBLY
- Best regards, Rob

**Comment by tim-moody** [comment]
That's a very nice, concise description. Thanks for taking the time to make a video! My only comment would be to mention Raspberry Pi installations if you ever make more videos. Want to get involved in the project?

**Comment by holta** [comment]
> Perhaps you want to add it to the YouTube video's highlighted?
>
> https://youtu.be/-FAw27pOBLY

@crconsultants we need more videos like this!

Any chance you can give me a quick call in the coming week?

Just FYI I unfortunately cannot add this video to [IIAB's YouTube channel](https://www.youtube.com/channel/UC0cBGCxr_WPBPa3IqPVEe3g), because the "SAVE" button yields error message **"This action is turned off for content made for kids"** as explained here: https://support.google.com/youtube/answer/9632097

Which is of course ridiculous (as IIAB is made for kids!) so we need to try to work this out, and might also use your help here (-:

**Comment by holta** [comment]
@crconsultants 

> Any chance you can give me a quick call in the coming week?

holt @ unleashkids DOT org if poss, Thanks Rob!
